frontend/integration-tests/protractor.conf.ts

```diff
@@ -25,6 +25,7 @@ export const testName = `test-${Math.random()
   .replace(/[^a-z]+/g, '')
   .substr(0, 5)}`;
 export const screenshotsDir = 'gui_test_screenshots';
+export const firstElementByTestId = (id: string) => $(`[data-test-id=${id}]`);
```

Comment:
I'd move this down with the other exported utilities near line ~230

Reply:
`ID` should be all caps per console naming conventions

```suggestion
export const firstElementByTestID = (id: string) => $(`[data-test-id=${id}]`);
```

Reply:
fixed